### DISCUSSION TITLE
How can I remap ideajoin

### DISCUSSION THREAD

**Discussion by mslalith** [JetBrains/ideavim#474] [Q&A]
Hi, I'm using ideavim with Android Studio

I had already have <S-j> mapped to move down
Now I want to remap ideajoin to some other key.

Is there any way to remap this?

**Comment by TonicArtos** [comment]
try `nmap J <Action>(EditorJoinLines)`

**Reply by citizenmatt** [reply]
You could also use `nnoremap J <C-J>`. This will map `J` to the original action assigned to `<C-J>`, without remapping it to the new action.
